plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -285,122 +309,117 @@ private function init_feature_definitions(): void {
 
 		$legacy_features = array(
 			'analytics'                   => array(
-				'name'               => __( 'Analytics', 'woocommerce' ),
-				'description'        => __( 'Enable WooCommerce Analytics', 'woocommerce' ),
-				'option_key'         => Analytics::TOGGLE_OPTION_NAME,
-				'is_experimental'    => false,
-				'enabled_by_default' => true,
-				'disable_ui'         => false,
-				'is_legacy'          => true,
+				'name'                         => __( 'Analytics', 'woocommerce' ),
+				'description'                  => __( 'Enable WooCommerce Analytics', 'woocommerce' ),
+				'option_key'                   => Analytics::TOGGLE_OPTION_NAME,
+				'is_experimental'              => false,
+				'enabled_by_default'           => true,
+				'disable_ui'                   => false,
+				'skip_compatibility_checks'    => true,
```

Comment:
For the cases where we are replacing

```
'is_legacy' => true,
```

with (` 'compatible'` being the default)
```
skip_compatibility_checks'    => true,
'default_plugin_compatibility' => 'compatible',
```

Does it make sense to have `skip_compatibility_checks'    => true,` in the replacement (just double-checking not specific concerns or whatever)?

Reply:
That's a fair question. `skip_compatibility_checks = true` is not required, but it was the "direct" translation of the prior `is_legacy`. My plan is, once the changes in this PR are in, to ask the teams that contributed those features what they think the default compatibility should be and whether `skip_compatibility_checks` should be set (for cases where the default compatibility is set to `incompatible`).

Right now, that both flags are set is a bit redundant, but not problematic.